.github/workflows/manual_update_master.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         script: |
           cd websites/info_grid/
           git log -1
-          git clean -f
-          git pull
+          git fetch
+          git reset --hard origin/master
           source ~/.cargo/env
```

Comment:
this is not needed, mdbook is statically built, and shouldn't depend on having the rust toolchain?

Reply:
As discussed with Thabet, what about this for each branch, note that we would adjust the branch name at the git checkout line:

websites/www2/info_grid/
          git log -1
          git fetch
          git checkout development
          git reset --hard origin/development
          mdbook build -d docs

***

Also we could remove the following line as Thabet proposes:

source ~/.cargo/env